pkg/container-collection/options.go

```diff
@@ -788,6 +799,63 @@ func WithOCIConfigEnrichment() ContainerCollectionOption {
 	}
 }
 
+// WithOCIConfigForInitialContainer enriches initial containers with the OCI config.
+// It assumes that the initial containers are already populated in the ContainerCollection
+func WithOCIConfigForInitialContainer() ContainerCollectionOption {
+	return func(cc *ContainerCollection) error {
+		for _, container := range cc.initialContainers {
+			ppid := 0
+			sPath := filepath.Join(host.HostProcFs, fmt.Sprintf("%d", container.ContainerPid()), "status")
+			sFile, err := os.Open(sPath)
+			if err != nil {
+				log.Errorf("OCIConfig enricher: failed to open status file for container %s: %s", container.Runtime.ContainerID, err)
+				continue
+			}
+			defer sFile.Close()
+
+			// Read the status file to get the parent pid
+			reader := bufio.NewReader(sFile)
+			for {
+				line, err := reader.ReadString('\n')
+				if err != nil {
+					log.Errorf("OCIConfig enricher: failed to read status file for container %s: %s", container.Runtime.ContainerID, err)
+					break
+				}
+				if !strings.HasPrefix(line, "PPid:") {
+					continue
+				}
+				ppidStr := strings.TrimPrefix(line, "PPid:\t")
+				ppidStr = strings.TrimSuffix(ppidStr, "\n")
+				ppid, err = strconv.Atoi(ppidStr)
+				if err != nil {
+					log.Warnf("cannot parse PPid from /proc/PID/status: %s", err)
+				}
+				break
+			}
+			target, err := os.Readlink(filepath.Join(host.HostProcFs, fmt.Sprintf("%d", ppid), "cwd"))
+			if err != nil {
+				log.Errorf("OCIConfig enricher: failed to read cwd symlink for container %s: %s", container.Runtime.ContainerID, err)
+				continue
+			}
+			bPath := filepath.Join(host.HostRoot, target)
```

Comment:
```suggestion
			bPath := securejoin.SecureJoin(host.HostRoot, target)
```
But I think the `securejoin.SecureJoin(host.HostRoot, ...)` has to be done after appending `container.Runtime.ContainerID` and `config.json` with `filepath.Join`.